syntax = "proto3";

message School {
  int32 id = 1;
  string name = 2;
  string email = 3;
  string phone_number = 4;
  string address = 5;
}

message Class {
  string class_name = 1;
}

message SchoolsRequest{}

message SchoolsResponse{
  repeated School schools = 1;
}

message GetClassRequest{}

message GetClassResponse{
  repeated Class classes = 1;
}

service CommonService{
  rpc Schools(SchoolsRequest) returns (SchoolsResponse);
  rpc GetClass(GetClassRequest) returns (GetClassResponse);
}